src/coreclr/gc/gc.cpp

```diff
@@ -14605,6 +14755,8 @@ BOOL gc_heap::soh_try_fit (int gen_number,
 #ifdef USE_REGIONS
                 if (can_allocate)
                 {
+                    // disable for GC in coreclr, as we run into an assert there because
```

Comment:
changes in this method can be eliminated if you just merge with the latest.

Reply:
Right, done.